xrootd/xrdfs/file.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xrdfs
+
+import (
+	"go-hep.org/x/hep/xrootd/internal/xrdenc"
+)
+
+// File implements access to a content and meta information of file over XRootD.
+type File interface {
+	Compression() *FileCompression // Compression returns the compression info.
+	Info() *EntryStat              // Info returns the stat info.
+	Handle() FileHandle            // Handle returns the file handle.
+}
+
+// FileHandle is the file handle, which should be treated as opaque data.
+type FileHandle [4]byte
+
+// FileCompression holds the compression parameters such as the page size and the type of compression.
+type FileCompression struct {
+	PageSize int32
+	Type     [4]byte
```

Comment:
please document this exported field as well. (I guess it's the name of the compression algorithm?)